.github/workflows/create-system-test-docker-base-images.yml

```diff
@@ -42,10 +46,33 @@ jobs:
         AzureDevopsBuildId: "${{ github.event.inputs.azdo_build_id }}"
 
     - uses: ./.github/actions/create-system-test-docker-base-images
+      if: ${{ false }}  # disable for now. TODO enable again
       name: 'Create system test docker images'
       with:
         artifacts_path: "${{steps.assets.outputs.artifacts_path}}"
         package_version: "${{steps.versions.outputs.version}}"
         lib_waf_version: "${{steps.versions.outputs.lib_waf_version}}"
         waf_rules_version: "${{steps.versions.outputs.waf_rules_version}}"
-        github_token: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - uses: ./.github/actions/create-system-test-docker-base-images-linux
+      name: 'Create system test linux docker images'
+      with:
+        artifacts_path: "${{steps.assets.outputs.artifacts_path}}"
+        package_version: "${{steps.versions.outputs.version}}"
+        commit_sha: "${{github.event.inputs.commit_sha}}"
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Prune registry
+      uses: vlaurin/action-ghcr-prune@0a539594d122b915e71c59733a5b115bfaaf5d52 #v0.5.0
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        organization: Datadog
+        container: dd-trace-dotnet/apm-library-dotnet
+        keep-younger-than: 7 # days
```

Comment:
Is this safe? 🤔 I can't remember if we use these images for anything other than system tests? This isn't pruning the lib injection images is it?